### PR TITLE
Update CKEditor 5 dependencies and code to recent changes in CKEditor 5 codebase

### LINKED ISSUE #89: Update CKEditor 5 dependencies and code to recent CKEditor 5 changes

### PR DESCRIPTION
**Pull request by ma2ciek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Other: Updated CKEditor 5 dependencies and code to recent changes in CKEditor 5 codebase. The editor data can be provided now by `config.initialData` or `data` properties, but providing it using both properties will cause an error. Closes #89.

---

### Additional information

*For example – encountered issues, assumptions you had to make, other affected tickets, etc.*

### CODE REVIEW

File: src/ckeditor/ckeditor.component.ts [hunk 1]
```diff
@@ -216,20 +216,23 @@ export class CKEditorComponent implements AfterViewInit, OnDestroy, ControlValue
...
 	 */
-	private createEditor(): Promise<any> {
+	private createEditor(): Promise<void> {
 		const element = document.createElement( this.tagName );
 		this.editorElement = element;
 
-		// Do not use the `editor.setData()` here because of the issue in the collaboration mode (#6).
-		// Instead set data to the element on which the editor will be later initialized.
-		element.innerHTML = this.data;
+		let config = this.config;
```
Comment (pomek) [inline src/ckeditor/ckeditor.component.ts]:
```js
confg config = Object.create( {}, this.config, { 
  initialData: this.data || ''
} );
```

Wouldn't be better?

Comment (pomek) [reply src/ckeditor/ckeditor.component.ts]: If not, feel free to self-merge. I have no objections 👍

Comment (ma2ciek) [reply src/ckeditor/ckeditor.component.ts]: I was thinking about a scenario when someone passes data using the `initialData`, leaving the `data` property empty. But maybe I can handle both cases here with `initialData: this.data || config.initialData || ''`.

Comment (ma2ciek) [reply src/ckeditor/ckeditor.component.ts]: Maybe I'll also add a simple test for it.

**Pull request merged**